### ISSUE TITLE
bug: core card: tab navigation on table card skips card content section

### ISSUE THREAD

**Issue by Seamoo13** [SAP/fundamental-ngx#3547]
#### Is this a bug, enhancement, or feature request?

bug

#### Briefly describe your proposal.

When using keyboard navigation on the table card, the focus skips the table content (table cells). Tab navigation should be able to access each section of a card and be able to use arrow navigation through each row of content
env: https://fundamental-ngx.netlify.app/#/core/card

#### If this is a bug, please provide steps for reproducing it.

1. go to the table card component
2. click on the header
3. hit the 'tab' key and pay attention to the focus

![Peek 2020-10-05 10-59](https://user-images.githubusercontent.com/47522152/95054010-e1367d00-06f9-11eb-9cee-1811b6e41abf.gif)
